pkg/consensus/consensus.go

```diff
@@ -299,7 +300,8 @@ func (s *service) getKeyPair(pubs []crypto.PublicKey) (int, crypto.PrivateKey, c
 
 		key, err := keys.NEP2Decrypt(acc.EncryptedWIF, s.Config.Wallet.Password)
 		if err != nil {
-			continue
+			s.log.Fatal("can't unlock account", zap.String("address", address.Uint160ToString(sh)))
+			break
```

Comment:
We should fail as early as possible, this one is a configuration error, we should get it like immediately upon node startup. I'd propose moving wallet opening to the network's `NewServer` and checking the password there. The node just won't start at all this way and this is what we want when it's obviously not configured properly.

Of course this more late check also makes sense, but only as an additional one, not the primary.

Reply:
Though returning an error from `consensus.NewService` probably also is fine.

Reply:
Fixed.